node/packages/aws-lambda-otel-extension/internal/otel-extension-internal-node/wrapper.js

```diff
@@ -18,6 +18,7 @@ const wrapHandler = (handlerFunction) => {
   let requestStartTime;
   let responseStartTime;
   let currentInvocationId = 0;
+  global.invokeStartDate = new Date().getTime();
```

Comment:
I am not a fan of this. @medikoo Do you know of a more reliable way to get the invocation time from the external part of the extension? I need to get an invocation time for the request that has a timestamp before the first log statement 🤔

Reply:
> I need to get an invocation time for the request that has a timestamp before the first log statement 🤔

Logs in lambda can be written before first invocation (during the initialization phase), e.g. with provisioned concurrency, it means that logs can be written minutes before the first invocation (or even with no invocations at all, when lambda won't get invoked).

So the question is, do we want to track the start time of each invocation? Or start time of lambda initialization? This code point is not right for any of that :)

Reply:
That is okay if the initialization logs are published before the request. We basically just want to get a timestamp as close to the invocation time as possible.

Reply:
Ok, so it's about the timestamp that will mark the start of the invocation and which will be updated at the start of every next invocation.

In this case, this is the right codepoint: https://github.com/serverless/console/blob/547d0d377b7fd1f3b9f536b3139faf27b4861b50/node/packages/aws-lambda-otel-extension/internal/otel-extension-internal-node/wrapper.js#L58

I would definitely not expose it as global, but e.g., we can expose it as `EvalError.$serverlessIncocationStart` (it's sort of hacky, but not intrusive. We're adding a property on a global object, which is for sure not used in any code) and `Date.now()` is more natural way to get current timestamp

Reply:
Awesome, I moved off of the global variable to the `EvalError` variable and switched both date instances to use `Date.now()`.

Both `global` and `EvalError` feel hacky to me but at least `EvalError` is far less likely to be used in customer code so I like this suggestion 😎